BrainLILODrv.cpp

```diff
@@ -207,14 +277,20 @@ static bool doLinux()
     if (iter != models.end())
     {
         model = iter->second;
-    } else {
+    }
+    else
+    {
         outputDebugMessage(L"BrainLILO: internal model name %s is unknown, falling back to u-boot.bin", model.c_str());
         model = L"u-boot.bin";
     }
 
     fn += model;
     outputDebugMessage(L"BrainLILO: opening Bootloader file: %s", fn.c_str());
 
+    brain_gen = SelectGen(model);
+    outputDebugMessage(L"BrainLILO: This is %d generation Brain", brain_gen);
```

Comment:
(nits) Please start the text next to `BrainLILO:` lower case.
```suggestion
    outputDebugMessage(L"BrainLILO: this is %d generation Brain", brain_gen);
```

(IMO) Printing enumerated numbers doesn't make sense to me. It's better off printing like `BrainLILO: generation 1` and `BrainLILO: generation 2 or later` with if-else.